client/src/modules/dashboard/shared/components/EventList.tsx

```diff
@@ -26,38 +34,54 @@ export const EventList = ({ events, emptyText, title }: Props) => {
       {events.length > 0 ? (
         <Grid gap="2em">
           {events.map(({ canceled, id, invite_only, name }) => (
-            <Flex justifyContent="space-between" key={id}>
-              <LinkButton href={`/dashboard/events/${id}`}>{name}</LinkButton>
```

Comment:
Using grid doesn't provide the responsive layout desired from grid system,

Reading the issue, you can create something similar to the menu in the header for this button, here is the chakra docs 

https://chakra-ui.com/docs/components/menu#menuitem-as-a-link

Reply:
@Sboonny I don't have anything specific in mind

Reply:
Thank you for the suggestions, we will adjust our implementation.